.github/workflows/request_api.yml

```diff
@@ -0,0 +1,89 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python Request API
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  repository_dispatch:
+    types: [ deploy_success ]
```

Comment:
I brought this up yesterday as well, but still checking again -- are the deploys configured in such a way that it picks up staging URL on staging deploys and that of prod's on prod deploys?